demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
what about a "link editor" (thanks to @LexZhukov for the idea), so that in the view mode it's `html`, and in the edit mode this is two inputs, where the first one is a link address and the second one link name?
```
[http://google.com] [Google] -> <a href="http://google.com">Google</a>
```
What do you think?